spec/03-plugins/23-rate-limiting/04-access_spec.lua

```diff
@@ -12,6 +12,17 @@ local fmt = string.format
 local proxy_client = helpers.proxy_client
 
 
+local function it_with_retry(desc, test)
+  return it(desc, function(...)
+    local ok = pcall(test, ...)
+    if not ok then
+      ngx.sleep(2)
```

Comment:
This sleep should not be necessary. If a test failed due to a minute-flip, it should be possible to execute it immediately (and it's actually better to _not_ sleep, because it gives it more time to run within the following minute).